samples/dymaptic.GeoBlazor.Core.Sample.Maui/MauiProgram.cs

```diff
@@ -27,16 +27,16 @@ public static MauiApp CreateMauiApp()
         builder.Services.AddScoped<HttpClient>();
         builder.Services.AddGeoBlazor();
 
-        string? apiKey = Preferences.Get("ArcGISApiKey", null);
+        builder.Configuration.AddInMemoryCollection();
+
+        var a = Assembly.GetExecutingAssembly();
```

Comment:
you may want to add a comment on why you are doing it this way, what needs to be in the secret (or where to find our documentation on it), and also re-name the variable from "a"

Reply:
Good point. I stole this from testing from #151 , but I will probably just merge that work in here, since it is small.